azure-kusto-data/example.js

```diff
@@ -6,39 +6,43 @@ const KustoConnectionStringBuilder = require("azure-kusto-data").KustoConnection
 const ClientRequestProperties = require("azure-kusto-data").ClientRequestProperties;
 const uuidv4 = require("uuid/v4");
 
-let clusterName = "";
-let username = "username";
-let password = "password";
+const clusterConectionString = "https://<name>.kusto.windows.net";
+const database = "";
+const table = "";
 
-const kcs = KustoConnectionStringBuilder.withAadUserPasswordAuthentication(`https://${clusterName}.kusto.windows.net`, username, password);
+const kcs = KustoConnectionStringBuilder.withAadDeviceAuthentication(clusterConectionString);
```

Comment:
is the customer able to provide authority?

Reply:
yes